app/src/components/Courses/CourseListCard.tsx

```diff
@@ -17,7 +17,43 @@ import { mdiStar } from "@mdi/js";
  * @param {Course} course The course to be displayed
  * @returns HTML Element
  */
-export const CourseListCard = ({ course}: { course: Course },) => {
+export const CourseListCard = ({ course }: { course: Course }) => {
+  const [imageSrc, setImageSrc] = useState<string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const maxTitleLength = 20;
+  //only load coursecard if ti exists
```

Comment:
_Minor_ detail, but a white space between the comment slashes and the first word would be swell (also an upper case 'O'). Likewise on line 29 😁